Plausible/Sampleable.lean

```diff
@@ -262,6 +296,9 @@ instance [SampleableExt α] [SampleableExt β] : SampleableExt ((_ : α) × β)
     let p ← prodOf sample sample
     return ⟨p.fst, p.snd⟩
   interp s := ⟨interp s.fst, interp s.snd⟩
+  proxyExpr? := none -- did not understand the difference with products
```

Comment:
To be clear, this is `Sigma` not `Prod`.

Reply:
Thanks. Indeed the documentation said so but I missed this. Will fix.